<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgressDialogClass</class>
 <widget class="QDialog" name="ProgressDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="mainwidget_" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#mainwidget_{background:white; border:1px solid #B4B4B4;}
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="30,0,146,0,44">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>1</number>
      </property>
      <item>
       <widget class="ltp::base::DialogTitleWidget" name="titleWidget_" native="true"/>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="styleSheet">
         <string notr="true">border-bottom: 1px solid #D2D2D2;</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="contentWidget_" native="true">
        <property name="styleSheet">
         <string notr="true">background:white;</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>19</number>
         </property>
         <property name="leftMargin">
          <number>30</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>30</number>
         </property>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>62</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="contentLabel_">
           <property name="styleSheet">
            <string notr="true">font:14px SIMHEI;</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar_">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>14</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>14</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QProgressBar{background-color:#E6E6E6; text-align:right; margin-right:40px; font:14px Arial;} 
QProgressBar::chunk{background-color:#0064C8;}</string>
           </property>
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>62</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="styleSheet">
         <string notr="true">border-bottom: 1px solid #D2D2D2;</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="bottomWidget_" native="true">
        <property name="styleSheet">
         <string notr="true">background:#F5F5F5;</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>287</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="ltp::base::DialogButton" name="cancelButton_">
           <property name="minimumSize">
            <size>
             <width>86</width>
             <height>24</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>取消</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ltp::base::DialogTitleWidget</class>
   <extends>QWidget</extends>
   <header>base/dialogtitlewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ltp::base::DialogButton</class>
   <extends>QPushButton</extends>
   <header>pushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
